src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureBlobStorage.cs

```diff
@@ -254,5 +190,42 @@ private static string GetBlobName(string grainType, GrainReference grainId)
                 throw new InconsistentStateException($"Blob storage condition not Satisfied.  BlobName: {blob.Name}, Container: {blob.Container?.Name}, CurrentETag: {currentETag}", "Unknown", currentETag, ex);
             }
         }
+
+        public void Participate(ISiloLifecycle lifecycle)
+        {
+            lifecycle.Subscribe(this.options.InitStage, Init);
+        }
+
+        /// <summary> Initialization function for this storage provider. </summary>
+        /// <see cref="IProvider.Init"/>
+        private async Task Init(CancellationToken ct)
+        {
+            try
+            {
+                this.jsonSettings = OrleansJsonSerializer.UpdateSerializerSettings(OrleansJsonSerializer.GetDefaultSerializerSettings(this.typeResolver, this.grainFactory), this.options.UseFullAssemblyNames, this.options.IndentJson, this.options.TypeNameHandling);
```

Comment:
add more provider name specific logging to improve debugging experience. See #4032 as an example. And also this is what we did for legacy provider initialization logging too